mappings/net/minecraft/item/Item.mapping

```diff
@@ -10,24 +39,81 @@ CLASS none/aeo net/minecraft/item/Item
 		FIELD h efficiency F
 		FIELD i damage F
 		FIELD j enchantability I
+		METHOD <init> (Ljava/lang/String;IIIFFI)V
+			ARG 0 miningLevel
+			ARG 1 durability
+			ARG 2 efficiency
+			ARG 3 damage
+			ARG 4 enchantability
 		METHOD a getDurability ()I
 		METHOD b getEfficiency ()F
 		METHOD c getWeaponDamage ()F
 		METHOD d getMiningLevel ()I
 		METHOD e getEnchantability ()I
 		METHOD f getRepairItem ()Lnone/aeo;
+	CLASS none/aeo$10
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
+	CLASS none/aeo$11
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
+	CLASS none/aeo$12
+		METHOD a getProperty (Lnone/aeq;Lnone/aiu;Lnone/sn;)F
+			ARG 0 stack
+			ARG 1 world
+			ARG 2 user
+	CLASS none/aeo$13
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
+	CLASS none/aeo$14
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
+	CLASS none/aeo$15
+		METHOD a getProperty (Lnone/aeq;Lnone/aiu;Lnone/sn;)F
+			ARG 0 stack
+			ARG 1 world
+			ARG 2 user
+	CLASS none/aeo$16
+		METHOD a getProperty (Lnone/aeq;Lnone/aiu;Lnone/sn;)F
+			ARG 0 stack
+			ARG 1 world
+			ARG 2 user
+	CLASS none/aeo$17
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
+	CLASS none/aeo$18
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
+	CLASS none/aeo$19
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
+	CLASS none/aeo$20
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
+	CLASS none/aeo$21
+		METHOD a apply (Lnone/aeq;)Ljava/lang/String;
+			ARG 0 stack
 	FIELD a BLOCK_ITEM_MAP Ljava/util/Map;
+	FIELD b GETTER_DAMAGED Lnone/aer;
```

Comment:
I went with `GETTER_X` for the reason mentioned above. That has been our naming scheme for a lot of things, like `MODIFIER_X` and `ItemX`